core/state/statedb.go

```diff
@@ -206,9 +206,10 @@ func (s *StateDB) AddLog(log *types.Log) {
 	s.logSize++
 }
 
-func (s *StateDB) GetLogs(hash common.Hash, blockHash common.Hash) []*types.Log {
+func (s *StateDB) GetLogs(hash common.Hash, blockNumber uint64, blockHash common.Hash) []*types.Log {
```

Comment:
Please add a docstring , e.g. 
```suggestion
// GetLogs returns the logs matching the specified transaction hash, and annotates them with the given blocknumber/blockhash 
func (s *StateDB) GetLogs(hash common.Hash, blockNumber uint64, blockHash common.Hash) []*types.Log {
```